pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -227,6 +227,9 @@ def hpc_save(self, folderpath: str, logger):
 
         model.on_hpc_save(checkpoint)
 
+        if self.trainer.accelerator_backend:
+            checkpoint = self.trainer.accelerator_backend.on_save(checkpoint)
```

Comment:
isn't accelerator backend always set? even on cpu?

Reply:
I think it isnt set when `auto_lr_find` is used.